composer.json

```diff
@@ -71,5 +72,13 @@
         "sort-packages": true
     },
     "prefer-stable": true,
-    "minimum-stability": "dev"
+    "minimum-stability": "dev",
+    "scripts": {
+        "post-update-cmd": "Google_Task_Composer::cleanup"
+    },
+    "extra": {
+        "google/apiclient-services": [
```

Comment:
Hele tohle je nějakej config pro toho apiclienta? Nejde to předávat nějak jinak? Tady bych to totiž asi nikdy nehledal.

Reply:
přesně nevím, postupoval jsem podle jejich dokumentace. Proč je to v extra nevím

Reply:
Aha, už koukám, že to je jen kvůli promazání závislostí a není to tedy žádná konfigurace toho klienta.